src/seo/base-seo.ts

```diff
@@ -1,6 +1,8 @@
-const seo = {
+import { Metadata } from 'next';
+
+const baseSeo: Metadata = {
+  metadataBase: new URL('https://elixir-cloud-aai-github-io.vercel.app'),
```

Comment:
This is the correct URL: https://elixir-cloud.dcc.sib.swiss/

Reply:
Resolved